progly.tex

```diff
@@ -35,22 +35,21 @@ \section{Programmable Storage}
 API~\cite{balmin:sigmod2012-clydesdale} to improve performance.
 
 Rather than relying on storage systems to evolve \emph{or} applications to
-change, a hybrid approach that embraces interface instability allows maximum
-flexibility, so long as it does not impose an unmanageable burden on
-developers.
+change, a hybrid approach that embraces interface instability without placing
+an unmanageable burden on developers is ``programmable storage".
 
-\subsection{The Malacology Approach}
+\subsection{The Programmable Storage Approach}
```

Comment:
I'm concerned about this title because its (2) Programmable Storage (2.1) The Programmable Storage Approach or does that make sense and i'm tired